### ISSUE TITLE
Incorrect documentation of snap on mosquitto.org

### ISSUE THREAD

**Issue by pjsg** [eclipse/mosquitto#2077]
The page: https://mosquitto.org/documentation/using-the-snap/ says:

*On Linux systems that have snap support, Mosquitto can be installed from the graphical software installer, or with `snap install mosquitto`.*

*After installing the Mosquitto snap, the Mosquitto broker will be running with the default configuration, which means it is listening for connections on port 1883.*

However, this is not true any more. If you follow these instructions, then you end up without a working service on port 1883. I understand that this was a deliberate breaking change -- but it probably means that the vast majority of auto updating snap users are now dead in the water.

For anybody suffering from this problem (`Client <unknown> disconnected, not authorised.`, `Connection error: Connection Refused: not authorised.
`) the solution is:

```
sudo snap refresh mosquitto --channel=1.6/stable
```

**Comment by ralight** [comment]
Repeating my comment from https://github.com/eclipse/mosquitto/issues/2074:

@pjsg you're right that the snap documentation wasn't updated to point users to the need to configure authentication, I've now fixed that, thank you.

The default configuration of the 2.0 snap package not authorising any connections is an oversight which I didn't catch because it seems I have only tested with different variations of configuration file, but not without a custom configuration file. It seems as though the same is true for all of the people who have been using the 2.0 snap over the past two months as well.

Sorry for the disruption.

The snap is now fixed to allow unauthenticated access from the local machine only, as per the default configuration. If you want to allow access from other computers you will need to configure a listener and an authentication method, either a password file, authentication plugin, or setting `allow_anonmous true` if that is what you wish.
